### ISSUE TITLE
Security Issue: Set proper content type for serving JS files in the static webserver

### ISSUE THREAD

**Issue by mrnicegyu11** [ITISFoundation/osparc-simcore#2784]
Note: This is related to the new security headers that we want to enable. They are currently only set on aws-staging, so you wont see this issue on aws-prod atm.

Reproduce:
Turn on DevTools in your browser, navigate to `staging.osparc.io/s4l`. You will see that the download of `boot.js` fails (due to the novel security headers). 

The problem & solution is detailed here: https://stackoverflow.com/a/53062522

tl;dr: The Javascript file is served in a REST response with `content-type | text/plain; charset=utf-8`, however, content type should be `application/javascript;charset=utf-8` for javascript files.

**Comment by GitHK** [comment]
Are you really sure about this? From what I can see it is served as `content-type: application/javascript`.

```shell
curl -v https://staging.osparc.io/s4l/boot.js
*   Trying 34.194.64.221:443...
* TCP_NODELAY set
* Connected to staging.osparc.io (34.194.64.221) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* successfully set certificate verify locations:
*   CAfile: /etc/ssl/certs/ca-certificates.crt
  CApath: /etc/ssl/certs
* TLSv1.3 (OUT), TLS handshake, Client hello (1):
* TLSv1.3 (IN), TLS handshake, Server hello (2):
* TLSv1.3 (IN), TLS handshake, Encrypted Extensions (8):
* TLSv1.3 (IN), TLS handshake, Certificate (11):
* TLSv1.3 (IN), TLS handshake, CERT verify (15):
* TLSv1.3 (IN), TLS handshake, Finished (20):
* TLSv1.3 (OUT), TLS change cipher, Change cipher spec (1):
* TLSv1.3 (OUT), TLS handshake, Finished (20):
* SSL connection using TLSv1.3 / TLS_AES_128_GCM_SHA256
* ALPN, server accepted to use h2
* Server certificate:
*  subject: CN=staging.osparc.io
*  start date: Jan 28 01:40:15 2022 GMT
*  expire date: Apr 28 01:40:14 2022 GMT
*  subjectAltName: host "staging.osparc.io" matched cert's "staging.osparc.io"
*  issuer: C=US; O=Let's Encrypt; CN=R3
*  SSL certificate verify ok.
* Using HTTP2, server supports multi-use
* Connection state changed (HTTP/2 confirmed)
* Copying HTTP/2 data in stream buffer to connection buffer after upgrade: len=0
* Using Stream ID: 1 (easy handle 0x556e55193e10)
> GET /s4l/boot.js HTTP/2
> Host: staging.osparc.io
> user-agent: curl/7.68.0
> accept: */*
>
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* Connection state changed (MAX_CONCURRENT_STREAMS == 250)!
< HTTP/2 200
< accept-ranges: bytes
< cache-control: public, max-age=86400
< content-type: application/javascript
< date: Thu, 03 Feb 2022 10:36:37 GMT
< last-modified: Tue, 01 Feb 2022 17:32:15 GMT
< strict-transport-security: max-age=15768001; includeSubDomains
< vary: Accept-Encoding
< vary: Accept-Encoding
< x-frame-options: SAMEORIGIN
< x-xss-protection: 1; mode=block
<
```

**Comment by mrnicegyu11** [comment]
Firefox on `https://staging.osparc.io/s4l` gives
![image](https://user-images.githubusercontent.com/8209087/152326949-4b6361d6-dbe9-4769-b062-fa6f92d750ec.png)

**Comment by mrnicegyu11** [comment]
The issue is due to 404 for `boot.js` being served with `text/plain` attribute. This does not need to be fixed. Closing.
